docs/primary-submission-walkthrough.md

```diff
@@ -1,13 +1,24 @@
 ## Pre-requisites
 
-### Set-up a service account
+### Generate an Authenticating JWT
 
-You must have a valid token identifying your service in order to use the Ingest API.
+You must have a valid [JWT](https://jwt.io/introduction/) to identifying your service in order to use the Ingest API. A JWT can be created using the [HCA-CLI](https://github.com/HumanCellAtlas/dcp-cli). install with:
 
-The DCP uses Google service accounts for authenticating service-to-service requests. To request a service account and associated RSA key-pair, create an issue in [Fusillade](https://github.com/HumanCellAtlas/fusillade), the DCP's auth service.
+`$ pip install hca`
 
-Use [this](https://github.com/HumanCellAtlas/dcplib/blob/master/dcplib/security/dcp_service_account.py) library with your key file to sign a JWT token identifying your service.
+Once installed authenticate your service by running the command and following the instructions:
 
+`$ hca dss login`
+
+If you're authenticating on a remote machine use:
+
+`$ hca dss login --remote`
```

Comment:
What do you mean "remote machine"?   Remote to what?  Remote with respect to the DCP?  Remote from my office?

Reply:
I modified statement to be more specific

Reply:
👍